lua/refactoring/debug/print_var.lua

```diff
@@ -4,12 +4,47 @@ local Region = require("refactoring.region")
 local refactor_setup = require("refactoring.tasks.refactor_setup")
 local post_refactor = require("refactoring.tasks.post_refactor")
 local lsp_utils = require("refactoring.lsp_utils")
+local ts_utils = require("nvim-treesitter.ts_utils")
+local parsers = require("nvim-treesitter.parsers")
 local debug_utils = require("refactoring.debug.debug_utils")
 local ensure_code_gen = require("refactoring.tasks.ensure_code_gen")
 local get_select_input = require("refactoring.get_select_input")
 
 local function get_variable()
     local variable_region = Region:from_current_selection()
+    if
+        variable_region.start_col == 0
```

Comment:
>  the main issue I'm having here is this is a one and done as soon as one thing is highlighted, then this won't work.
@teddylear  I think it's fine. We support this feature in both the `normal` and 'visual` modes. The `print_var` in `normal` mode helps us call this feature faster (not have to switch from `normal` to `visual` to print the var). In case the `normal` mode doesn't work, the user can try in `visual` mode.

Reply:
Sounds good, I'll merge as is and then make a follow up ticket to change that. @thuan1412 Thanks again for the PR.